appoetry1/Flows/LoginFlow.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  LoginFlow.swift
+//  appoetry1
+//
+//  Created by Kristaps Brēmers on 25.02.19.
+//  Copyright © 2019. g. Chili. All rights reserved.
+//
+
+import UIKit
+
+class LoginFlow:  FlowController {
+    func start() {
+        guard let vc = loginViewController else {
+            fatalError("Could not get login vc")
+        }
+        self.rootController.pushViewController(vc, animated: true)
+    }
+    
+    let rootController = UINavigationController()
```

Comment:
It can be initialised with root controller straight away, and it actually must do that! No UINavigationController()